pkg/controllers/update.go

```diff
@@ -329,7 +329,7 @@ func isUpdateAvailableViaWinget(updateVersion string) bool {
 	utils.LogDebug(strOut)
 
 	// Ex: `Doppler.doppler 3.63.1 3.64.0 winget`
-	re := regexp.MustCompile(fmt.Sprintf(`%s\s+%s\s+%s\s+winget`, wingetPackageId, strings.TrimPrefix(version.ProgramVersion, "v"), strings.TrimPrefix(updateVersion, "v")))
+	re := regexp.MustCompile(fmt.Sprintf(`%s\s+\d+\.\d+\.\d+\s+%s\s+winget`, wingetPackageId, strings.TrimPrefix(updateVersion, "v")))
```

Comment:
check: Why do we need to do this? The strict check seems more desirable

Reply:
This fixes an edge case where the version of the CLI binary performing the update is different than the binary version previously installed by winget. It's mostly just to aid with testing, though I didn't see a downside from the change. If a user does hit this, we probably still want to allow them to update.